cd/auto-generate/auto-generate-controllers.sh

```diff
@@ -79,6 +82,14 @@ else
   echo "auto-generate-controllers.sh][INFO] ACK runtime version for new controllers will be $ACK_RUNTIME_VERSION"
 fi
 
+GO_MODULE_VERSION=$(grep -E "^go [0-9]+\.[0-9]+$" go.mod | cut -d " " -f2)
```

Comment:
ah, you are referring to the Go *language* version, not the Go *module* version, right?

Reply:
In other words, this variable name should be `GO_VERSION` or `GO_LANG_VERSION`, yes?

Reply:
yes, I will update the variable name.